dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -478,7 +477,7 @@ private void convertRegistryIdsToRegistries() {
             }
             Arrays.stream(arr).forEach(id -> {
                 if (registries.stream().noneMatch(reg -> reg.getId().equals(id))) {
-                    RegistryConfig registryConfig = new RegistryConfig();
+                    RegistryConfig registryConfig = new RegistryConfig(RegistryConfig.NO_AVAILABLE);

```

Comment:
I don't think we can give RegistryConfig a default value `RegistryConfig.NO_AVAILABLE`, instead we should leave it empty. We will try to `refresh` this RegistryConfig later, if the user fails to set the right item for RegistryConfig, it will receive an Exception then when checking `isValid`. But if we give it this default value, it will always pass `isValid` and does register URL to Registry Center which can be unexpected behaviour.